### ISSUE TITLE
Test case not specifying UTF-8

### ISSUE THREAD

**Issue by jordig100** [3MFConsortium/test_suites#85]
In all examples it shows that the UTF-8 is specified. But there is not a case with UTF-8 not being specified, and this is a case to be supported but not testest, according to the spec:

https://github.com/3MFConsortium/spec_core/blob/master/3MF%20Core%20Specification.md#232-xml-usage

**Comment by JimZuber** [comment]
Good Catch! Will be trivial to modify an existing Core test case to omit the encoding attribute from the XML declaration. Will try to do this as part of the displacements project (added a reminder in my project notes).
